bin/devil-bot-rust-cdk.ts

```diff
@@ -7,3 +7,7 @@ const app = new cdk.App();
 new DevilBotRustCdkStack(app, 'DevilBotRustCdkStack', {
     env: { region: app.node.tryGetContext('region') || 'us-east-1' },
 });
+// Dev stage
+new DevilBotRustCdkStack(app, 'DevilBotRustCdkStackDev', {
+    env: { region: app.node.tryGetContext('region') || 'us-east-1' },
```

Comment:
Nit, but we should update the region on these (don't do this now, just a note for later).